happi/cli.py

```diff
@@ -165,10 +165,18 @@ def happi_cli(args):
     elif args.cmd == 'edit':
         logger.debug('Starting edit block')
         device = client.find_device(name=args.name)
+        is_invalid_field = False
         for edit in args.edits:
             field, value = edit.split('=', 1)
-            logger.info(f'Setting {args.name}.{field} = {value}')
-            setattr(device, field, value)
+            try:
+                getattr(device, field)
+                logger.info(f'Setting {args.name}.{field} = {value}')
+                setattr(device, field, value)
+            except AttributeError as ex:
```

Comment:
A bare exception may be better here, as we could catch anything that might go wrong:

```suggestion
            except Exception:
```

Reply:
Ok sounds good, thank you

Reply:
In this case, yes - having the error message is very useful, but catching other exceptions would be good, too. Would you mind changing it to include the error message here?

Reply:
Ohh yeah sorry about that!! I added it back.